nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ConfigureProxy.java

```diff
@@ -57,6 +55,17 @@ public void setPortProperty(String port) {
 
     @Override
     public void execute() throws BuildException {
+        log("Proxy configuration", Project.MSG_INFO);
+        
+        // Prevent proxy configuration on CI environments (Travis, GitHub actions)
+        String isCIEnvironment = System.getenv("CI");
```

Comment:
This is not correct check in my opinion and once the `configureproxy` is idenpotent - e.g. #2210, you will not need this check.

Reply:
I uptated it removing the CI check